Compile error

Build like this:

SET CGO_ENABLED=0
SET GOOS=windows
SET GOARCH=386
go build

Got an error.
.\connector.go:45:25: constant 9223372036854775807 overflows uint.\connector.go:45:25: constant 9223372036854775807 overflows uint

it should be like this in line 45: 
cfg.ReconnectAttempts = math.MaxUint32

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,6 @@
+# 0.4.2 | [Pull request](https://github.com/furdarius/rabbitroutine/pull/7)
+- Fix compile error on Windows `GOOS=windows GOARCH=386 go build` ([issue](https://github.com/furdarius/rabbitroutine/issues/7))
+
 # 0.4.1 | [Pull request](https://github.com/furdarius/rabbitroutine/pull/6)
 - Data race fixed on c.conn when connection error occurs.
 
@@ -22,3 +25,4 @@
 - Dial and DialConfig were added. DialConfig used to configure RabbitMQ connection settings.
 - Config stores only reconnect options.
 - Start replaced with Dial.
+

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,4 +1,4 @@
-# 0.4.2 | [Pull request](https://github.com/furdarius/rabbitroutine/pull/7)
+# 0.4.2 | [Pull request](https://github.com/furdarius/rabbitroutine/pull/8)
 - Fix compile error on Windows `GOOS=windows GOARCH=386 go build` ([issue](https://github.com/furdarius/rabbitroutine/issues/7))
 
 # 0.4.1 | [Pull request](https://github.com/furdarius/rabbitroutine/pull/6)

--- a/connector.go
+++ b/connector.go
@@ -42,7 +42,7 @@ type Connector struct {
 // NewConnector return a new instance of Connector.
 func NewConnector(cfg Config) *Connector {
 	if cfg.ReconnectAttempts == 0 {
-		cfg.ReconnectAttempts = math.MaxInt64
+		cfg.ReconnectAttempts = math.MaxUint32
 	}
 
 	return &Connector{
@@ -308,7 +308,6 @@ func (c *Connector) DialConfig(ctx context.Context, url string, config amqp.Conf
 		// In the case of connection problems,
 		// we will get an error from closeCh
 		closeCh := c.conn.NotifyClose(make(chan *amqp.Error, 1))
-
 
 		// After context cancellation we must wait for finishing of connBroadcast to avoid data race on c.conn.
 		var wg sync.WaitGroup

